s2fft/utils/healpix_ffts.py

```diff
@@ -1,20 +1,20 @@
-from jax import jit, vmap
+from functools import partial
 
 import numpy as np
 import jax.numpy as jnp
-import torch
-from s2fft.sampling import s2_samples as samples
-from functools import partial
-from s2fft.utils.jax_primitive import BasePrimitive, register_primitive
 import jaxlib.mlir.ir as ir
-from s2fft_lib import _s2fft
+import torch
+from jax import jit, vmap
+from jax.lib import xla_client
 from jaxlib.hlo_helpers import custom_call
-from jax.core import  ShapedArray
-from typing import Tuple
-from jax import ShapeDtypeStruct
+
 # did not find promote_dtypes_complex outside _src
 from jax._src.numpy.util import promote_dtypes_complex
 
+from s2fft.utils.jax_primitive import register_primitive
+from s2fft.sampling import s2_samples as samples
+from s2fft_lib import _s2fft
+
 
 def spectral_folding(fm: np.ndarray, nphi: int, L: int) -> np.ndarray:
```

Comment:
I have coded the spectral folding ... however it is not exported
It is included inside the healpix FFT
Tell me if that is good for you